README.md

```diff
@@ -1,96 +1,43 @@
 # inlets-on-fly
 
-inlets-on-fly automates the task of creating an [inlets Pro](https://inlets.dev) exit-server (tunnel server) on the [fly.io](https://fly.io) platform.
+inlets-on-fly automates the task of creating an [inlets Pro](https://inlets.dev) HTTP exit-server (tunnel server) on the [fly.io](https://fly.io) platform.
```

Comment:
I used this to create a TCP exit server previously, so that I could expose SSH. Having TCP would make it useful for inlets-operator too.